go/admin_types.go

```diff
@@ -15,3 +15,27 @@ type SessionInfo struct {
 	RaftMembers       []MemberInfo `json:"raftMembers"`
 	ConnectedToLeader bool         `json:"connectedToLeader"`
 }
+
+type SessionLogResponse struct {
```

Comment:
`AdminGetSessionLogResponse` perhaps?

Reply:
Agreed, I could also move it to `admin_getsessionlog.go`